core/risk/margins_calculation.go

```diff
@@ -116,7 +116,7 @@ func (e *Engine) calculateMargins(m events.Margin, markPrice *num.Uint, rf types
 				err       error
 			)
 			if auction {
-				exitPrice = e.ob.GetIndicativePrice()
+				noExit = true
```

Comment:
I'm not sure about this. If we can get an accurate exit price during an auction, why would we not use it?

Reply:
Ah, that makes sense. Although, doing things this way, there might be an incongruity WRT opening auctions. IIRC, the margin requirements for orders during opening auctions were supposed to be based on the order price (if we don't have an indicative price - so until we have uncrossing orders on the book), and after that, the margin will be based on the indicative price.

I'll check the spec if that's mentioned anywhere, but if so, we might need to update that.

Reply:
It's always meant to be vwap for orders now in auctions, though that might be on the `cosmicelevator` spec branch.